docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  ```
+- Checkout chore branch
+  ```bash
+  git checkout -b update-prerequisites
+  ```
+- Check to see `Cargo.toml` is refering to the latest updated prerequisite crates
```

Comment:
At this point nothing was updated yet, why would that be the case? Or does this mean checking that there is no diff in the tree? I am not sure what this step is for
```suggestion
- Check that `Cargo.toml` refers to the latest versions of the updated prerequisite crates
```

Reply:
Removed this.